src/sdk/object_sdk.js

```diff
@@ -194,7 +194,7 @@ class ObjectSDK {
         const { bucket } = await bucket_namespace_cache.get_with_cache({ sdk: this, name });
         const policy_info = {
             s3_policy: bucket.s3_policy,
-            system_owner: bucket.system_owner,
+            system_owner: bucket.system_owner, // in NC NSFS we don't have this value
```

Comment:
minor: we might add this to bucketspace_fs in the future, so I would say -
```suggestion
            system_owner: bucket.system_owner, // note that bucketspace_fs currently doesn't return system_owner
```